app/models/host/managed.rb

```diff
@@ -985,7 +995,7 @@ def ensure_associations
   end
 
   def set_default_user
-    return if self.owner_type.present? && !OWNER_TYPES.include?(self.owner_type)
+    return if self.owner_type.present? && !OWNER_TYPES.include?(self.owner_type) # to propagate invalid value to validations
     self.owner_type ||= 'User'
```

Comment:
not sure I follow, we set it only if owner type is present which means it was specified